### ISSUE TITLE
Error loading electron (electron ^5.0.8 and electron-edge-js ^12.0.1)

### ISSUE THREAD

**Issue by andrearx** [agracio/electron-edge-js#50]
Hi
I'm unable to start my app next updating electron ^5.0.8 and electron-edge-js ^12.0.1
The same project with electron 4.x works perfectly (and always electron-edge-js ^12.0.1)

```
npm ERR! code ELIFECYCLE
npm ERR! errno 3221226505
npm ERR! xxx@7.0.0 start: 'electron .'
npm ERR! Exit status 3221226505
npm ERR!
npm ERR! Failed at the xxx@7.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

(I watched the issue #32 but I red that was solved in 10.11.1)

Any suggestions?

Thanks,
Andrea

**Comment by ukenny** [comment]
I've spent the last day trying to figure this out since I am using `electron-edge-js` at my job and this is what fixed it (assuming you project is a .NET Framework project): Adding ` nodeIntegrationInWorker: true` in the `webPrefences` when instantiating the `BrowserWindow` in the `main.js`. I hope that solves it!

**Comment by andrearx** [comment]
@ukenny unfortunately I already have that configuration and not works.
(I am using a  .NET Framework project)

**Comment by andrearx** [comment]
Issue fixed with v12.4.0 :ok_hand:
